extension/content.js

```diff
@@ -328,6 +368,11 @@ document.addEventListener('DOMContentLoaded', () => {
 			if (isCommit()) {
 				addPatchDiffLinks();
 			}
+
+			if (isPRFiles()) {
```

Comment:
You could break the `isPR()` and `isCommit()` checks above into smaller pieces using a new one like `isPRCommit()` and then you could combine checks.